### PR TITLE
create 

### PR DESCRIPTION
**Pull request by qspidersam**

### CODE REVIEW

File: ComcastCRMGUIFramework1/src/main/java/com/comcast/crm/objectrepositoryutility/Createproductpage.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.comcast.crm.objectrepositoryutility;
+
+public class Createproductpage {
+	//code1
+	//code2
+
+}
```
Comment (supriyaaaaaaaaaa) [inline ComcastCRMGUIFramework1/src/main/java/com/comcast/crm/objectrepositoryutility/Createproductpage.java:7]: change the xpath

**Review by supriyaaaaaaaaaa — COMMENTED** [review]
added fewc few comment

File: ComcastCRMGUIFramework1/src/main/java/com/comcast/crm/objectrepositoryutility/Createproductpage.java (hunk 1 above)
Comment (qspidersam) [reply ComcastCRMGUIFramework1/src/main/java/com/comcast/crm/objectrepositoryutility/Createproductpage.java:7]: fixed

**Comment by supriyaaaaaaaaaa** [comment]
merged

**Pull request merged**